taxonomy/forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+Taxonomy forms
+"""
+
+from django.forms import (
+    ModelForm,
+    TextInput,
+    CheckboxSelectMultiple,
+    RadioSelect,
+)
+
+from .models import Vocabulary
+
+
+class VocabularyForm(ModelForm):
```

Comment:
The same way Django does all its localization:

`from django.utils.translation import ugettext_lazy`

It's often imported as `_`, as in:

`from django.utils.translation import ugettext_lazy as _`

Then you handle it with `.po` files as usual in Python.